.github/workflows/ci.yaml

```diff
@@ -446,6 +446,77 @@ jobs:
         run: |
           poetry run pytest tests/ -vvv -n 2 ${PYTEST_OPTS}  ${{ matrix.PYTEST_OPTS }}
 
+  min-btc-support:
+    name: Test BTC version 25.0 ${{ matrix.name }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    env:
+      RUST_PROFILE: release  # Has to match the one in the compile step
+      PYTEST_OPTS: --timeout=1200 --force-flaky
+    needs:
+      - compile
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - NAME: clang
+            CFG: compile-clang
+            TEST_DB_PROVIDER: sqlite3
+            COMPILER: clang
+            TEST_NETWORK: regtest
```

Comment:
nit: put the BITCOIN_VERSION as a matrix value instead of repeatedly putting it into the definition below.

Reply:
Fixed by adding MIN_BTC_VERSION for the step.